src/cutadapt/cli.py

```diff
@@ -296,6 +296,10 @@ def get_argument_parser() -> ArgumentParser:
         metavar="ERRORS",
         help="Discard reads whose expected number of errors (computed "
             "from quality values) exceeds ERRORS.")
+    group.add_argument("--max-average-error-rate", "--max-er", type=float, default=None,
+        metavar="ERROR_RATE",
+        help="Discard reads whose expected average error rate (computed "
+             "from quality values divided by length) exceeds ERROR_RATE")
```

Comment:
"computed from quality values dividid by length" doesn’t make so much sense. Perhaps just refer to `--max-expected-errors`: "(computed as for `--max-expected-errors`, but divided by length)" or so